File: December 03/dec3.py
def buildingHieght(arr):
    count = 1
    for i in range(1,len(arr)):
        if arr[i-1] < arr[i]:
            count+=1
    return count

def main():
    n = int(input())
    Harr = []
    for i in range(n):
        h = int(input())
        Harr.append(h)
    print(buildingHieght(Harr))
main()

File: December 03/dec3.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
    int n;
    cout << "Enter the size: ";
    cin >> n;

    vector<int> arr;

    for (int i = 0; i < n; i++)
    {
        int a;
        cin >> a;
        arr.push_back(a);
    }
    
    int count=1;

    for(int i=1; i<n; i++)
    {
        if(arr[i]>arr[i-1])
        {
            count++;
        }
    }
    
    
    cout<<count;
}
